static/src/js/components/FormFields/RadioList/RadioList.js

```diff
@@ -31,22 +25,8 @@ export const RadioList = ({
   return (
     <div className="radio-list">
       {
-        Children.map(children, (child) => {
-          const { props, type } = child
-
-          if (type !== Radio && type !== AccessMethodRadio) return null
-
-          return React.cloneElement(
-            child,
-            {
-              onChange: onPropsChange,
-              // Disabling the rule, but we shouldn't be using Children
-              // https://react.dev/reference/react/Children
-              // eslint-disable-next-line react/prop-types
-              checked: selected === props.value
-            }
-          )
-        })
+        radioList.length === 0 ? null
```

Comment:
Do you need this conditional here? If radioList is empty, `.map()` won't do anything